xtimeutil/remapper.py

```diff
@@ -199,13 +208,45 @@ def get_outgoing_time_axis_info(self):
         x.update(attrs)
         return x
 
+    def mean(self, data):
+        if isinstance(data, xr.DataArray):
```

Comment:
You could also make this a "check-raise" statement with:

```python
if not isinstance(data, xr.DataArray):
    raise NotImplementedError('mean currently only works with xarray DataArrays')
```

Then the remaining code in the function need not be indented, but this is not a necessary change.